### ISSUE TITLE
can't join quests by clicking via quest page

### ISSUE THREAD

**Issue by Ritnio** [RolePlayGateway/roleplaygateway#95]
I clicked on a quest in the MV and it gave me the page you see in the screenshot, but nothing else happens. If a character is joined already I can click them, but I can't click any of the characters under "Join With"

No idea what the problem is... I'm sure I could just join in chat, but still worth pointing out I guess.

![](https://cdn.discordapp.com/attachments/363418857123086346/476984863120162816/unknown.png)

**Comment by colearkenach** [comment]
The images are only being displayed as images. Inside the source code, there is an "alt=" which I'm not sure what it does? Is this what is supposed to be controlling what accepts and joins the quest @martindale 

`alt="Participate with Ashley J. "Ash" Williams"`

**Comment by Ritnio** [comment]
Who are you?
How did you get my email?
Why are you emailing me?

Get Outlook for Android<https://aka.ms/ghei36>

________________________________

**Comment by LawOfTheLand** [comment]
I managed it just fine. Darien Blackdagger is now signed up for quest number 1538, shooting some guns at On Target Gun Range and Store. I had to go in character as him and used the `/quest accept 1538` method.

**Comment by Ritnio** [comment]
Ok. I know what this is, confused me cuz phone went off about a dude named Jeremy

Get Outlook for Android<https://aka.ms/ghei36>

________________________________

**Comment by martindale** [comment]
> The images are only being displayed as images. Inside the source code, there is an "alt=" which I'm not sure what it does? Is this what is supposed to be controlling what accepts and joins the quest @martindale
> 
> `alt="Participate with Ashley J. "Ash" Williams"`

No, this stuff isn't built out yet — it'll be shipped before 2.0

**Comment by lostamongtrees** [comment]
Thank you @Ritnio and @LawOfTheLand ! Closing this issue - compiled with #51 for full notes on quest functioning/nonfunctioning

**Comment by lostamongtrees** [comment]
repening still cant from page
